composer.json

```diff
@@ -21,16 +21,16 @@
         }
     ],
     "require": {
-        "php": "^8.1",
+        "php": "^8.3",
```

Comment:
Please keep support for the older versions also. 
Ref - https://github.com/freshbitsweb/laratables/blob/master/composer.json#L6